tests/README.rst

```diff
@@ -0,0 +1,8 @@
+Python Infrastructure Tests
+===========================
+
+Python Infrastructure tests are run on Docker containers with Kitchen Salt (https://kitchen.saltstack.com), testinfra (https://github.com/philpep/testinfra/), and pytest (https://pytest.org).
+
+1. Install Bundler (`gem install bundler`)
+2. Install gems in Gemfile (`bundle install`)
```

Comment:
Similarly I ran into a system vs local issue here and used `bundle install --path vendor/bundle` to keep from installing into system paths.

Reply:
Fixed.